spyder/plugins/completion/kite/utils/status.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""Kite status functions."""
+
+# Standard library imports
+import logging
+import os
+import os.path as osp
+import subprocess
+import sys
+
+# Third-party imports
+import psutil
+
+logger = logging.getLogger(__name__)
+
+
+def check_if_kite_installed():
+    """Detect if kite is installed and return the installation path."""
+    path = ''
+    if os.name == 'nt':
+        path = 'C:\\Program Files\\Kite\\kited.exe'
+    elif sys.platform.startswith('linux'):
+        path = osp.expanduser('~/.local/share/kite/kited')
+    elif sys.platform == 'darwin':
+        path = locate_kite_darwin()
+    return osp.exists(osp.realpath(path)), path
```

Comment:
How could we handle custom installations on specific locations?

Reply:
I'm unsure about this